src/main/java/com/axibase/tsd/client/SelfSignedTrustManager.java

```diff
@@ -31,4 +34,24 @@ public void checkClientTrusted(X509Certificate[] chain,
                                    String authType) throws CertificateException {
         //No implementation here
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
+    }
+
+    @Override
+    protected void finalize() throws Throwable {
+    }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') com.axibase.tsd.client.SelfSignedTrustManager.finalize() is empty and should be deleted [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=findbugs%3AFI_EMPTY)